src/main/java/nextstep/qna/domain/Answer.java

```diff
@@ -72,8 +68,23 @@ public void toQuestion(Question question) {
         this.question = question;
     }
 
+    public void delete(NsUser loginUser) throws CannotDeleteException {
+        authorityValidation(loginUser);
+        this.deleted = true;
+    }
```

Comment:
```suggestion
    public DeleteHistory delete(NsUser loginUser) throws CannotDeleteException {
        authorityValidation(loginUser);
        this.deleted = true;
        // DeleteHistory 반환
    }
```

이러면 삭제와 이력을 한번에 처리할 수 있습니다.